pos_payment_terminal/static/tests/pos_payment_terminal_tests.js

```diff
@@ -0,0 +1,272 @@
+odoo.define("pos_payment_terminal.pos_payment_terminal_tests", function (require) {
+    "use strict";
+
+    /* global QUnit */
+
+    var OCAPaymentTerminal = require("pos_payment_terminal.payment");
+
+    require("web.test_utils");
+    const Session = require("web.Session");
+    const makePosTestEnv = require("point_of_sale.test_env");
```

Comment:
I need to check if these `require` items are still enough for the test to actually run